staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/values.go

```diff
@@ -58,7 +58,7 @@ func UnstructuredToVal(unstructured interface{}, schema *structuralschema.Struct
 	if schema.Type == "object" {
 		m, ok := unstructured.(map[string]interface{})
 		if !ok {
-			return types.NewErr("invalid data, expected map[string]interface{} to match the provided schema with type=object")
+			return types.NewErr("invalid data, expected a map format to match the provided schema with type=object")
```

Comment:
"invalid data, expected a map for the provided schema with type=object but got a {schema.Type}" ?

(same for below array error)

Reply:
By the time reaching here, schema.Type has already been declared as an `object'. I have rephrased the message without returning {schema.Type}